apps/client/src/app/components/header/header.component.html

```diff
@@ -1,4 +1,9 @@
 <mat-toolbar class="p-0">
+  <ng-container *ngIf="!user">
+    <a [routerLink]="['/']" class="no-min-width px-2" mat-button>
+      <gf-logo></gf-logo>
+    </a>
+  </ng-container>
```

Comment:
Suggestion: Shows logo when user is not signed in yet.

Reply:
It's not so easy, because the logo should not be displayed on the landing page (big logo below)